src/components/AddExercise.vue

```diff
@@ -459,4 +535,17 @@ button {
   border: 1px hidden rgba(32, 32, 32, 0);
   border-radius: 5px;
 }
+
+.btn-check:focus + .btn-outline-danger,
+.btn-outline-danger:focus + .btn-outline-danger {
```

Comment:
there is the `shadow-none` class in bootstrap for that

Reply:
done